src/h3lib/lib/faceijk.c

```diff
@@ -32,8 +32,9 @@
 #include "latLng.h"
 #include "vec3d.h"
 
-/** square root of 7 */
-#define M_SQRT7 2.6457513110645905905016157536392604257102
+/** square root of 7 and inverse square root of 7 */
+#define M_SQRT7  2.6457513110645905905016157536392604257102
+#define M_RSQRT7 0.37796447300922722721451653623418006081576
```

Comment:
I think this is also going to fail the automatic formatting check

Reply:
thanks, just fixed